test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -130,6 +140,37 @@ def test_qdel_nagle_perf(self):
         self.logger.info(
             "qdel performance after setting manager: " + str(qdel_perf2))
 
-        # Check that the two timings are pretty close to each other
-        self.assertTrue((qdel_perf2 - qdel_perf) < 5,
-                        "qdel performance differs too much!")
+    @timeout(600)
+    def test_qsub_perf(self):
+        """
+        Test that qsub performance have improved when run
+        with -f option
+        """
+
+        # Restart server again
```

Comment:
Tests can run out of order, so you might not be restarting the server "again".  Maybe say "Restarting server.  See comment in test_qdel_nagle_perf"  Or something like that